src/util/git.ts

```diff
@@ -31,12 +34,58 @@ const didCheckoutFromMaster = async () => {
 
   if (!splitLastCheckoutMove) return FAILED_GET_LASTCHECKOUT
 
-  const isMaster = splitLastCheckoutMove[0].trim() === 'master'
+  return splitLastCheckoutMove[0].trim() === 'master'
+}
+
+const validateFiles = (changedFilesString: string, lessonId: number) => {
+  // 4 is js3 or Objects
+  if (lessonId > 4) return
```

Comment:
Are these the the same `lessonId` as stored in the db ? If so the ids for the firsrt four lessons are
js0 - 5
js1 - 2
js2 - 1
js3 - 4,
so this check will not work as intended.